.gitignore

```diff
@@ -12,3 +12,5 @@
 *.out
 
 .idea/
+.vscode/settings.json
+.vscode/launch.json
```

Comment:
Would it make more sense to exclude `.vscode` altogether? eg `.vscode/`

Reply:
Yeah, we can even take all of that out entirely but it's not going to hurt